rules/linux/persistence_kernel_driver_load_by_non_root.toml

```diff
@@ -0,0 +1,96 @@
+[metadata]
+creation_date = "2024/01/10"
+integration = ["auditd_manager"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2024/01/10"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects the loading of a Linux kernel module by a non-root user through system calls. Threat actors may leverage Linux
+kernel modules to load a rootkit on a system providing them with complete control and the ability to hide from security
+products. As other rules monitor for the addition of Linux kernel modules through system utilities or .ko files, this
```

Comment:
Curious if you would like to call out related rules in the note section something like
note = "### Related rules
 <Rule -1>
"
Just gives an overall picture to the user to enable all rules with kernel module monitoring.

Reply:
This will be done in the future investigation guide!